### PR TITLE
[la7] Add embedded urls, update tests

### PR DESCRIPTION
**Pull request by timendum**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

I've update the la7 extractor with new test cases, removing old invalid cases and accepting new urls.

### CODE REVIEW

File: youtube_dl/extractor/la7.py [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
+        embedded/la7[^/]*(?:&|\?)content=
     )(?P<id>.+)'''
 
     _TESTS = [{
         # 'src' is a plain URL
         'url': 'http://www.la7.it/crozza/video/inccool8-02-10-2015-163722',
         'md5': '8b613ffc0c4bf9b9e377169fc19c214c',
         'info_dict': {
-            'id': 'inccool8-02-10-2015-163722',
+            'id': '0_42j6wd36',
```
Comment (yan12125) [inline youtube_dl/extractor/la7.py]: It's better to keep the original ID for existing patterns, or --download-archive will be broken.

Comment (timendum) [reply youtube_dl/extractor/la7.py]: I did not change the id extraction, it changed in the Kaltura extractor.

**Comment by yan12125** [comment]
Sorry for not checking the ID carefully. You are correct.

For the "invalid" test URL, I found a replacement: http://tg.la7.it/repliche-tgla7?id=231700. Could you add the pattern back and update the test?

**Comment by timendum** [comment]
I can add the `_extract_urls` to check these urls, with embedded page, but the test you provided is no longer valid.

Should I push it anyway?

**Comment by yan12125** [comment]
Well, there's another - http://tg.la7.it/repliche-tgla7?id=233847

Better to add a comment that such URLs are valid for only a short time.

**Comment by timendum** [comment]
Now the extractor is capable of extracting video from these url, also added a new test case, with "match only".

Thanks for the input.

**Pull request closed without merging**